database/migrations/2022_10_10_175307_create_login_histories_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('login_history', function (Blueprint $table) {
+            $table->id();
+            $table->integer('user_id');
+            $table->timestamp('created_at');
+            $table->char('ip');
```

Comment:
а почему char? может varchar?
еще в ларавель есть хелпер для колонки с ip
```suggestion
            $table->ipAddress('ip');
```